src/apiCalls.ts

```diff
@@ -9,5 +11,11 @@ export const getPlaylist = (genre: string) => {
   return (
     fetch(`http://ws.audioscrobbler.com/2.0/?method=tag.gettoptracks&tag=${genre}&api_key=${process.env.REACT_APP_LASTFM_APIKEY}&limit=300&format=json`)
       .then(response => response.json())
+      .then(data => {
+        const CleanedGenreTrackData: CleanedAlbumTrack[] = data.tracks.track.map((song: AlbumTrack)  => {
+          return cleanGenreTrackData(song)
+        })
+        return randomizeSongs(CleanedGenreTrackData)
+      })
```

Comment:
Really like the extra checking with the interfaces that were made previously to ensure that the data that is being stored is used is the data we expect it to be! Very nice.